src/main/scala/uk/gov/hmrc/githubclient/ExtendedTeamService.scala

```diff
@@ -38,4 +40,20 @@ class ExtendedTeamService(client: ExtendedGitHubClient) extends TeamService(clie
 
     client.put(uri.toString(), params, headers, classOf[Team])
   }
+
+  def getExtendedRepositories(id: Int): util.List[ExtendedRepository] = {
```

Comment:
Why did we choose to return the java.util.List here?

Reply:
@cjwebb Good spot - originally I was overriding the base method, so was using the same signature, but now this is a new method it doesn't have to be a Java list, I'll update to a Scala list.

Reply:
@cjwebb Updated and pushed.